tests/integration/workspace_access/test_redash.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from datetime import timedelta
+
+from databricks.sdk.errors import NotFound
+from databricks.sdk.retries import retried
+from databricks.sdk.service import sql
+
+from databricks.labs.ucx.workspace_access import redash
+from databricks.labs.ucx.workspace_access.groups import GroupManager
+from databricks.labs.ucx.workspace_access.manager import PermissionManager
+from databricks.labs.ucx.workspace_access.redash import RedashPermissionsSupport
+
+logger = logging.getLogger(__name__)
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=13))
```

Comment:
no, 13 minutes is too much. i don't believe this test needs more than 5 minutes of timeout, otherwise it'll slow down the entire merge pipeline

Reply:
redash permissions are cached for up to 10 mins. It is usually faster but worst case it can be 10 mins. So after the rename operation it can take 10 mins until we can see the renamed group in the permissions. On top we have to leave some time to actually perform the rest of the test.

Reply:
added comments about this in the test

Reply:
updated